libs/blocks/locui-create/utils/constant.js

```diff
@@ -1,6 +1,11 @@
 export const LOCALES = 'locales';
 export const LOCALE_GROUPS = 'localegroups';
 export const URL_SEPARATOR_PATTERN = /[,\r\n]/g;
+export const API_BASE_URL = {
+  dev: 'https://14257-miloc-dev.adobeioruntime.net/api/v1/web/miloc-0.0.1',
```

Comment:
@raga-adbe-gh Could you please get these added in Milo config file as well? Once done, we will remove it and pick it from there